ide/coqide/idetop.ml

```diff
@@ -87,13 +87,19 @@ let ide_doc = ref None
 let get_doc () = Option.get !ide_doc
 let set_doc doc = ide_doc := Some doc
 
+(* Adds n spaces at the beggining of the string *)
+let wrap_string n s = String.make n ' ' ^ s
+
 let add ((((s,eid),(sid,verbose)),off),(line_nb,bol_pos)) =
   let doc = get_doc () in
   let open Loc in
   (* note: this won't yield correct values for bol_pos_last,
      but the debugger doesn't use that *)
   let loc = { (initial ToplevelInput) with bp=off; line_nb } in
-  let pa = Pcoq.Parsable.make ~loc (Gramlib.Stream.of_string s) in
+  let s_wrapped = wrap_string off s in
+  let r_stream = Gramlib.Stream.of_string s_wrapped in
+  let () = Gramlib.Stream.njunk off r_stream in
```

Comment:
We could alternatively add a new API `Stream.set_count`, but I'm not sure that allowing users to mess with that is a good idea, WDYT?

Reply:
Maybe just an optional offset count argument for `of_string`?

Reply:
Good idea, will push a commit with that and see how it looks.